pkg/internal/apis/certmanager/validation/certificate_test.go

```diff
@@ -25,7 +25,7 @@ import (
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	"k8s.io/apimachinery/pkg/util/validation/field"
 
-	cmapiv1alpha2 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1alpha2"
+	cmapiv1 "github.com/jetstack/cert-manager/pkg/apis/certmanager/v1"
```

Comment:
nit: cmapiv1 -> cmapi

Reply:
Done. Although it seems a bit fishy that we're importing and using the external API version in the internal version.